src/main/java/org/homeschoolpebt/app/submission/actions/DataRequiredInterceptor.java

```diff
@@ -0,0 +1,135 @@
+package org.homeschoolpebt.app.submission.actions;
```

Comment:
Nitpick:

Why did you use the `submission/actions` dir structure? Since this is a spring interceptor, my first thought was it would be in an `interceptor` dir.

Reply:
Oops, it was because I initially tried using an action for this, but failed. I'll move it.